Doc/user_manual/six.tex

```diff
@@ -742,6 +834,9 @@ \subsubsection{Beam Position Monitor} \label{BPM}
 Usual SixTrack units are used. Any number of BPM elements can be used but
 the names must differ. 
 
+
+
+
 \subsubsection{Other element types}
 
 Some other elements, such as dipole edge (24), solenoid (25), multipole RF kicks ($\pm$26, $\pm$27, $\pm$28) are accepted by SixTrack, but they are not currently supported by the development team or tested for correctness.
```

Comment:
The multipoles should probably be removed from the "unsupported" list, as they are now tested and documented!